extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -548,6 +551,20 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
             function _isDisabledRequest(xhr?: XMLHttpRequestInstrumented, request?: Request | string, init?: RequestInit) {
                 let isDisabled = false;
                 let theUrl:string = ((!isString(request) ? ((request ||{}) as Request).url || "" : request as string) ||"").toLowerCase();
+
+                // check excludeRequestFromAutoTrackingRegex before stripping off any query string
+                arrForEach(_excludeRequestFromAutoTrackingRegex, (regex: string | RegExp) => {
+                    if (!isDisabled) {
+                        const regexp = new RegExp(regex);
```

Comment:
Looking at the documentation new RexExp(regExp), passing a Regular expression was only added in ES6 (so this won't work in older browsers, we should add an isString() check.

Reply:
I see, then should the parameter be "string[]" only?

Reply:
No, both is a nice option.

Reply:
Had a second thought.. I think it's good to provide both types as parameter thus provide users more flexibility right? I updated the checking 😬